rolf/rolf_main/models.py

```diff
@@ -180,6 +180,16 @@ def clone_flags(self, new_deployment):
                                 default=flag.default,
                                 description=flag.description)
 
+    def clear_old_pushes(self, keep=20):
+        deleted = 0
+        if self.push_set.all().count() < keep:
+            return deleted
+        for push in list(self.push_set.all().order_by(
```

Comment:
I'm not sure the explicit list cast is necessary. I think slicing the queryset may automatically coerce it to a list.